### PR TITLE
Adopt-A-Spec for JUGs #616

### PR DESCRIPTION
**Pull request by chrisguindon**
Signed-off-by: Christopher Guindon <chris.guindon@eclipse-foundation.org>

### CODE REVIEW

**Comment by chrisguindon** [comment]
This is a work in progress and it's not ready for review.

**Comment by shabnammayel** [comment]
-please change the title to "Adopt a Spec" Program for JUGs instead of Adopt a Specification for JUGS
_please separate the last item (note) under the sign up section and add * at the start of the sentence as outlined in this dochttps://docs.google.com/document/d/1nDRAJfpbM7fSdky2WYnI3sFyOv-T1xWALZJ8LH1PvLM/edit?ts=5e446539#
-In How JUG leader can help section: Ensure that the topic of the adopted Jakarta EE Specification is consistently discussed, e.g. dedicate a portion of the existing JUG meetups to the discussions related to the adopted Specification, monthly calls with all representatives of JUGs that are running the program in their groups. Please add "arrange" before "monthly calls with all representatives of JUGs that are running the program in their groups"

**Comment by shabnammayel** [comment]
@chrisguindon can you please remove the bulletin at the section of "How JUG leaders can help:"? thanks, I think this is good to go live. Can we add "Adopt A Spec" under the dropdown of the resources? thanks

**Comment by chrisguindon** [comment]
@shabnammayel I pushed out an update and the preview url will be updated in about 5 minutes.

If you give us a +1, I will make it go live!

**Comment by shabnammayel** [comment]
@ivargrimstad can you also please check the "pull request process" in the preview and provide feedback before we go live?

**Comment by chrisguindon** [comment]
Please note that I will remove the EF from the list of adopters once we get an approval to move forward.

**Comment by chrisguindon** [comment]
I pushed out an update to the PR to remove the Eclipse Foundation as an adopter. We should be ready to push this live if @ivargrimstad approves the content/pr!

File: config.toml [hunk 1]
```diff
@@ -109,32 +104,43 @@ enableRobotsTXT = true
...
 
+[[menu.main]]
+    name = "Compatible Products"
+    url = "/compatibility/"
+    weight = 5
+
+[[menu.main]]
+    name = "Adopt A Spec"
+    parent = "Resources"
+    url = "/adopt-a-spec/"
```
Comment (autumnfound) [inline config.toml]: This URL is broken

Comment (chrisguindon) [reply config.toml]: Good catch! Should be fixed now!

**Pull request merged**